### PR TITLE
Moved AssetAdmin variables into framework for reuse

### PR DESCRIPTION
**Pull request by clarkepaul**
Also delete a ui-theme font family which was causing linting issues trying to override the style.

This needs merging before AssetAdmin linting fix.

### CODE REVIEW

File: admin/client/src/styles/_variables.scss [hunk 1]
```diff
@@ -697,3 +699,36 @@ $cms-panel-xs: 60px;
...
+// Shadows, these need work to make consistent (used in AssetAdmin)
+$shadow-light: rgba(0, 0, 0, .2);
+$dropdown-shadow: 0 0 3px $shadow-light;
+$gallery-item-shadow: 0 1px 4px $gray-light;
+
+// Masks: typically behind modals, drag drop areas
+$page-mask-light: rgba(236, 239, 241, .9);
+
+
+// Variables for use within but not limited to the AssetAdmin module
```
Comment (hafriedlander) [inline admin/client/src/styles/_variables.scss]: Doc out of date

File: admin/client/src/styles/_variables.scss [hunk 2]
```diff
@@ -697,3 +699,36 @@ $cms-panel-xs: 60px;
...
+// Gallery item path to framework
+
+$framework-path:     "../../../../framework";
```
Comment (hafriedlander) [inline admin/client/src/styles/_variables.scss]: Seems needlessly complicated given we're in framework now :). `../../..`?

Comment (clarkepaul) [inline admin/client/src/styles/_variables.scss]: yup I saw that and removed that line, it was originally a straight copy from assetadmin but I've moved that line back.

Comment (clarkepaul) [inline admin/client/src/styles/_variables.scss]: Updated already

File: admin/client/src/styles/_variables.scss (hunk 1 above)
Comment (clarkepaul) [inline admin/client/src/styles/_variables.scss]: Whats out of date?

Comment (hafriedlander) [inline admin/client/src/styles/_variables.scss]: Why does it mention AssetAdmin module here? They're either appropriate for framework generally or not. I'd just say `// Gallery item sizes` or something.

Comment (clarkepaul) [inline admin/client/src/styles/_variables.scss]: They are currently only used in AssetAdmin but they should be generic enough for use elsewhere. The note is there just incase they don't get used again in framework and someone want to delete them—its a reminder to keep them even though they might look like they aren't used.

I can remove comment if you think its not needed?

Comment (clarkepaul) [inline admin/client/src/styles/_variables.scss]: FYI Improved comment

**Comment by hafriedlander** [comment]
Looks good to me. Will merge when green.

**Pull request merged**